### ISSUE TITLE
What rdf:type should DELETE notifications use

### ISSUE THREAD

**Issue by acoburn** [trellis-ldp/trellis#398]
@mjgiarlo brought this up on the mailing list. When a Trellis resource is deleted, the notification that is emitted has the `ldp:Resource` type. It seems that using the type of the deleted resource would make more sense. That is, if a basic container is deleted, then the type would be `ldp:BasicContainer`.

**Comment by ajs6f** [comment]
Agreed that the most specific type we can publish is the best choice, unless we have the option to publish more than one (not very familiar with AS), in which case we should publish _all_ that apply.

**Comment by acoburn** [comment]
I don't see why we couldn't publish all types. We already have a [method](https://github.com/trellis-ldp/trellis/blob/master/core/http/src/main/java/org/trellisldp/http/impl/HttpUtils.java#L96) that takes an LDP type and produces a list with all of the supertypes. We could just pass those values directly into the `SimpleEvent` constructor as we do [here](https://github.com/trellis-ldp/trellis/blob/master/core/http/src/main/java/org/trellisldp/http/impl/MutatingLdpHandler.java#L189-L190), but rather than passing a single value, we send the entire collection.

The `Event` interface is already structured to support a collection of type values.

**Comment by mjgiarlo** [comment]
That too would work for my purposes.
